compliance/lib/data_adapter.rego

```diff
@@ -43,11 +43,17 @@ process_name = name {
 
 process_args_list = args_list {
 	is_process
-	args_list := split(input.resource.command, " ")
+
+	# Gets all the process argument of the current process
+	# Expects format as the following: --<key><delimiter><value> for example: --config=a.json
+	# Supported delimiters are space and `=`
+	# Notice that the first argument is always the process path
+	args_list := regex.find_n("^\\S+|--\\S+=\\S+|--\\S+\\s\\S+", input.resource.command, -1)
```

Comment:
As my previous comment, The regex is pretty straightforward and documented in the comment.
It can be a bit complex to understand, but I assume that the reader has some experience with basic Rego, and can understand from the comments what it does. 

What would you change ?

Reply:
the comment is a nice addition, it still does not explain what's going on in this regex.

> I assume that the reader has some experience with basic Rego
1. we can't assume this
2. I would say I have some regex experience and I still had to go to some docs in order to even start understanding (I honestly don't).

> What would you change ?

I would find an alternative to regex.

Reply:
I am wondering about how to solve it.
I mean, Regex is the most common way to solve pattern matches, especially when the pattern is not trivial.

I get your point that it is more complex than it looks, but I really tried to simplify it and this is the best option that I found.